src/server/dis_read.c

```diff
@@ -566,6 +566,10 @@ dis_request_read(int sfds, struct batch_request *request)
 			break;
 
 #ifndef PBS_MOM
+		case PBS_BATCH_RegisterSched:
+			request->rq_ind.rq_register_sched.rq_name = disrst(sfds, &rc);
```

Comment:
Hope we are initialising rq_name. If not if disrst() returns EOF then Scheduler should have been killed and causes problems.

Reply:
Sorry didn't get you question completely...
If you are asking for initializing rq_name, its handled by alloc_br().
If you are asking for checking rc for EOF (or any other error) after disrst() then it handled down in code, after this switch statement.

Reply:
I was just asking about initialisation only which is handled by alloc_br() as we are doing memset inside. So we are good there.